unime/src/routes/badges/[id]/+page.svelte

```diff
@@ -178,19 +170,18 @@
         </p>
       </div>
 
-      <!-- Metadata (Table: Credential Subject) -->
+      <!-- Contents (Table: Credential Subject) -->
       <div>
-        <p class="pb-2 text-lg font-semibold text-black dark:text-white">{$LL.BADGE.DETAILS.METADATA()}</p>
+        <p class="pb-2 text-lg font-semibold text-black dark:text-white">{$LL.BADGE.DETAILS.CONTENTS()}</p>
         <div
           class="divide-y divide-solid divide-slate-200 rounded-xl border border-slate-200 bg-white dark:divide-slate-600 dark:border-slate-600 dark:bg-dark"
         >
           {#each Object.entries(entries) as entry}
             <div class="flex flex-col items-start px-4 py-[10px]">
               <p class="text-[13px]/[24px] font-medium text-slate-500">{entry[0]}</p>
-              <p class="w-full break-words text-[13px]/[24px] font-medium text-slate-800 dark:text-white">
-                <!-- TODO: this is a hacky way to display nested data, but also to remove enclosing quotes for regular strings -->
-                {JSON.stringify(entry[1]).slice(1, -1)}
-              </p>
+              <div class="w-full break-words text-[13px]/[24px] font-medium text-slate-800 dark:text-white">
+                <pre class="whitespace-pre-wrap [font-family:inherit]">{prettyPrint(entry[1])}</pre>
```

Comment:
Last thing then I will approve this PR, maybe call it printBadgeDetails(), or something like that so we now what it prints

Reply:
I think a function name should reflect what it does. The name is purposely generic since it is agnostic about its input, so I would not like to make the name too specific. The comment should add the necessary context, don't you think?

```ts
// TODO: this is a simple way to display any (potentially nested) data, since we don't have a proper UI design for it yet
const prettyPrint = (object: any): string => {
  return JSON.stringify(object, null, 2);
};
```